3scaleAdapter/cmd/main.go

```diff
@@ -78,6 +82,28 @@ func parseMetricsConfig() *metrics.Reporter {
 	return metrics.NewMetricsReporter(true, port)
 }
 
+func cacheConfigBuilder() *threescale.ProxyConfigCache {
+	cacheTTL := threescale.DefaultCacheTTL
+	cacheRefreshInterval := threescale.DefaultCacheRefreshBuffer
+	cacheEntriesMax := threescale.DefaultCacheLimit
+
+	if viper.IsSet("cache_ttl_seconds") {
+		ttl := time.Duration(viper.GetInt("cache_ttl_seconds"))
+		cacheTTL = time.Duration(time.Second * ttl)
+	}
+
+	if viper.IsSet("cache_refresh_seconds") {
+		refreshInterval := time.Duration(viper.GetInt("cache_refresh_seconds"))
+		cacheRefreshInterval = time.Duration(time.Second * refreshInterval)
```

Comment:
is there any relation between `cacheTTL` and `cacheRefreshInterval` we might want to enforce here such as very low `cacheTTL` when compared to `cacheRefreshInterval`?

Reply:
I tend to agree. We can just give some hints in the docs and do as the settings say without enforcing anything.